tests/shared/src/test/scala/org/http4s/EntityDecoderSuite.scala

```diff
@@ -410,7 +410,7 @@ class EntityDecoderSuite extends Http4sSuite {
       .tempFile(None, "foo", "bar", None)
       .use { tmpFile =>
         val response = mockServe(Request()) { req =>
-          req.decodeWith(EntityDecoder.textFile(tmpFile), strict = false) { _ =>
+          req.decodeWith(EntityDecoder.textFile[IO](tmpFile), strict = false) { _ =>
```

Comment:
This (and the similar changes) is only necessary for 2.13. Scala 3 seems to be able to infer this just fine.